src/main/java/com/weather/assignment/document/WeatherElements.java

```diff
@@ -0,0 +1,25 @@
+package com.weather.assignment.document;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.Date;
+
+
+@Document
+@Data
```

Comment:
Please don't use `@Data`. Instead, use the specific annotations that are required like `@Getter` and`@Setter`